src/components/home/modals/Receive.vue

```diff
@@ -110,30 +109,20 @@ export default {
     },
     computed: {
         receivedObject() {
-            if (this.invoice === '') {
-                return {
-                    protocol: PROTOCOL,
-                    api: 1,
-                    opc: OPC_ACCOUNT,
-                    address: this.address,
-                    amount: this.transferAmount
-                }
-            } else {
-                return {
-                    protocol: PROTOCOL,
-                    api: API_VERSION,
-                    opc: OPC_ACCOUNT,
-                    address: this.address,
-                    amount: this.transferAmount,
-                    invoice: this.invoice
-                }
+            return {
+                protocol: PROTOCOL,
+                api: API_VERSION,
```

Comment:
The `api` version should not be 3. If no invoice, it should show as 1; otherwise, it should show 2.